### ISSUE TITLE
Periodically save screenshots so that later we can make a gpx with this information

### ISSUE THREAD

**Issue by Intelligenzbestie** [nicoschmitt/necrobotvisualizer#173]
I am not sure how easy this is especially if nobody is currently logged in on the machine where the bot runs (e.g. I have a windows server in the basement that I use for personal botting but I only log on to this server using Remote Desktop once in a while).
BTW: This is the most appealing visualizer imho especially with the open street map usage and the beautiful icons, thanks for this great piece of work!

**Comment by niicojs** [comment]
Do you want the bot to log what it does? if so, you'll have to ask for this on the bot side.
Beeing able to create a GPX from the UI make sense (via clicking on waypoints maybe). I'll think about it.

**Comment by Intelligenzbestie** [comment]
Yes, exactly. Would make sense to create it from the UI. Thanks, best regards Richard
